docs/docs/platforms/openshift/OPENSHIFT_FAQ.md

```diff
@@ -0,0 +1,112 @@
+Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2":
+Forbidden: not usable by user or serviceaccount, provider "hostnetwork":
+Forbidden: not usable by user or serviceaccount, provider "hostaccess":
+Forbidden: not usable by user or serviceaccount, provider "node-exporter":
+Forbidden: not usable by user or serviceaccount, provider "efs-csi-scc":
+Forbidden: not usable by user or serviceaccount, provider "privileged":
+Forbidden: not usable by user or serviceaccount]
+```
+
+To fix this error, set `openshift.runWithRestrictedSCC` to true in Helm values.
+
+Another possible error in the events:
+
+```bash
+Error: container has runAsNonRoot and
+image will run as root (pod: "jira-0_atlassian", container: container)
+```
+
+This can happen if the ServiceAccount has been granted `nonroot` SCC privileges, but `runAsUser` has not been set in `securityContext`. To fix the error, use the following Helm values:
+
+```yaml
+jira:
+  securityContext:
+    fsGroup: 2001
+    runAsUser: 2001
+```
+
+See complete Helm values for the [Nonroot SCC](../OPENSHIFT/#__tabbed_1_2).
+
+## What's the Difference Between Restricted and Nonroot SCC
+
+From the security/compliance standpoint **Restricted SCC** enforces a more secure set of rules (or rather limitations). However, running DC containers as non root users does not expose any security threats either. The only two disadvantages is that the ServiceAccount needs to be granted nonroot SCC privileges, and there are more Helm values to be overridden. See: [Nonroot SCC](../OPENSHIFT/#__tabbed_1_2).
+
+:octicons-thumbsup-16: Running with a Restricted SCC is the recommended way to deploy Atlassian DC Helm charts. See: [Restricted SCC](../OPENSHIFT/#__tabbed_1_1)
+
+
+## NFS Permission Fixer Is Disabled, How Can I Make Shared Home Writable
+
+When running Atlassian DC containers with a restricted or nonroot SCC, it is not possible to run nfs-permission fixer container because it runs as root. You need to make sure the volume is writable either for an unprivileged restricted SCC user or a user defined in `securityContext` (with UID like 2001, 2002, 2003 depending on the product).
+
+If you run NFS server, you may either change permissions directly on the NFS server instance, or create a temporary ServiceAccount, grant it anyuid SCC privileges, create a pod that uses this ServiceAccount, mounts shared-home volume and recursively changes owner of the shared home directory. It can take longer for larger volumes. 
+
+## Can I Use a Custom Certificate and Key for My Route?
+
+By default all Routes that the Helm charts will create will use OpenShift Router default certificate and key. If you want to use a custom crt-key pair for an individual Route, create a [TLS secret](https://kubernetes.io/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret_tls/){.external} with `tls.crt`, `tls.key` and `ca.crt` (optional) in `data` and provide the secret name in Helm values:
```

Comment:
1. Add a comma after "By default"
2. Avoid the double-use of "will" by removing the first one (eg, "that the Helm charts create will use..."
3. Add a "the" between "will use" and "OpenShift Router".